lib/ex_unit_fixtures.ex

```diff
@@ -220,6 +220,24 @@ defmodule ExUnitFixtures do
     the module.
   """
   defmacro deffixture({name, info, params}, opts \\ [], body) do
+    create_name = :"fixture_create_#{name}"
+    dep_names = for {dep_name, _, _} <- params || [] do
+      dep_name
+    end
+
+    quote do
+      def unquote({name, info, params}), unquote(body)
+
+      ExUnitFixtures.register_fixture(
+        unquote(create_name), unquote(dep_names), quote(opts)
+      )
+    end
+  end
+
+  @doc """
+  Registers a fixture in the current module.
```

Comment:
Need some much better docs here - when to use it over deffixture, what it's advantages are?  Maybe some links between the two?

A mention in the moduledoc would probably be a good idea as well.